src/irc/IdentGenerator.ts

```diff
@@ -43,6 +44,13 @@ export class IdentGenerator {
         });
     }
 
+    static switchAroundMxid(user: MatrixUser) {
+        return user.host.split('.')
```

Comment:
Are IPv4s valid host addresses?
`1.0.0.127:jaller` :upside_down_face:

Reply:
Totally under the matrix spec :), you can even have a port :S